### ISSUE TITLE
Create redirection on post title change

### ISSUE THREAD

**Issue by cfenollosa** [cfenollosa/bashblog#108]
When a published blogpost is edited via `./bb.sh edit` and the title is changed, the post URL, which was associated with the old title, is lost.

Bashblog should ask the user if they would like to create a redirection, then create a simple HTML file with both a javascript redirection to the new URL and a link for those who have Javascript disabled.

Another trivial option is to add an .htaccess entry, however that is httpd-dependent and other httpd servers may ignore it. Since bashblog is platform agnostic, the best option, even if not 100% transparent, is the javascript+link redirection.

**Comment by Natfan** [comment]
Just to add, #128 is sort of relevant to this.

**Comment by cfenollosa** [comment]
Yes, thanks. This issue deals with problems generated inside bashblog, while #128 deals with external files. I've closed #128 but I'll leave this one open and think if it is worth dealing with this one (probably with softlinks)
